defaults/php-cs-fixer-risky.php

```diff
@@ -0,0 +1,4 @@
+<?php
+declare(strict_types=1);
```

Comment:
We follow PSR-12 style https://github.com/paysera/php-style-guide this means linebreak is needed after php opening tag. Please fix this in other places too.

Reply:
Done